packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/ai-agent-action/components/AIChatTab.tsx

```diff
@@ -0,0 +1,261 @@
+  transform: rotate(90deg);
+`;
+
+const StyledMessageContainer = styled.div`
+  width: 100%;
+`;
+
+type AIChatTabProps = {
+  agentId: string;
+};
+
+export const AIChatTab: React.FC<AIChatTabProps> = ({ agentId }) => {
+  const theme = useTheme();
+
+  const {
+    messages,
+    messagesLoading,
+    sendingMessage,
+    handleSendMessage,
+    input,
+    handleInputChange,
+  } = useAgentChat(agentId);
+
+  const isLoading = messagesLoading || sendingMessage;
+
+  return (
+    <StyledContainer>
+      {messages.length !== 0 && (
+        <StyledScrollWrapper componentInstanceId={agentId}>
+          {messages.map((msg) => (
+            <StyledMessageBubble
+              key={msg.id}
+              isUser={msg.role === AgentChatMessageRole.USER}
+            >
+              <StyledMessageRow>
+                {msg.role === AgentChatMessageRole.ASSISTANT && (
+                  <StyledAvatarContainer>
+                    <Avatar
+                      size="sm"
+                      placeholder="AI"
+                      Icon={IconSparkles}
+                      iconColor={theme.color.blue}
+                    />
+                  </StyledAvatarContainer>
+                )}
+                {msg.role === AgentChatMessageRole.USER && (
+                  <StyledAvatarContainer isUser>
+                    <Avatar size="sm" placeholder="U" type="rounded" />
+                  </StyledAvatarContainer>
+                )}
+                <StyledMessageContainer>
+                  <StyledMessageText
+                    isUser={msg.role === AgentChatMessageRole.USER}
+                  >
+                    {msg.role === AgentChatMessageRole.ASSISTANT &&
+                    !msg.content ? (
+                      <StyledDotsIconContainer>
+                        <StyledDotsIcon size={theme.icon.size.xl} />
+                      </StyledDotsIconContainer>
+                    ) : (
+                      msg.content
+                    )}
+                  </StyledMessageText>
+                  {msg.content && (
+                    <StyledMessageFooter className="message-footer">
+                      <span>
+                        {beautifyPastDateRelativeToNow(msg.createdAt)}
+                      </span>
+                      <LightCopyIconButton copyText={msg.content} />
+                    </StyledMessageFooter>
+                  )}
+                </StyledMessageContainer>
+              </StyledMessageRow>
+            </StyledMessageBubble>
+          ))}
+        </StyledScrollWrapper>
+      )}
+      {messages.length === 0 && !isLoading && (
+        <StyledEmptyState>
+          <StyledSparkleIcon>
+            <IconSparkles size={theme.icon.size.lg} color={theme.color.blue} />
+          </StyledSparkleIcon>
+          <StyledTitle>{t`Ask AI`}</StyledTitle>
+          <StyledDescription>
+            {t`Start a conversation to get instant insights, support, or updates
```

Comment:
I think the tab name and the design made sense for an agent that you launch for the command bar direction (which is the very next step so you necessarily have to throw it away, you can embed an agent behind feature flag in the command bar in this PR if you want). For the workflow tab I would opt for something simpler with a tab name called "Chat" and a chat bubble icon (e.g. IconMessage). 

For the text we should adapt it for something much more generic like "Start a conversation with your agent"